applications/app_test/Makefile

```diff
@@ -17,6 +17,6 @@ ifeq ($(CPU),native)
 endif
 
 # Main thread stack size
-CFLAGS += -DTHREAD_STACKSIZE_MAIN=4096
+CFLAGS += -DTHREAD_STACKSIZE_MAIN=4096*2
```

Comment:
why ? If not an oversight, I would make an other commit for that
And I would use 8192 instead of *2

Reply:
4096 is still enough finally.